src/main/kotlin/no/nav/familie/ef/sak/api/journalføring/JournalføringController.kt

```diff
@@ -34,11 +34,13 @@ class JournalføringController(private val journalføringService: Journalføring
     @PostMapping("/{journalpostId}/fullfor")
     fun fullførJournalpost(@PathVariable journalpostId: String,
                            @RequestBody journalføringRequest: JournalføringRequest,
-                           @RequestParam(name = "journalfoerendeEnhet") journalførendeEnhet: String): Ressurs<Long> {
+                           @RequestParam(name = "journalfoerendeEnhet") journalførendeEnhet: String,
```

Comment:
Hvorfor er journalførende enhet og navident lagt til som egne parametere i stedet for at JournalføringRequest er utviudet med disse feltene?

Reply:
Godt poeng, kanskje bake de inn i requestobjektet, ja.